lib/commands/crafter/add_office.rb

```diff
@@ -44,5 +36,15 @@ def run
         ""
       end
     end
+
+    private
+
+    def updated_crafter
```

Comment:
I think that we need to delete these counter now, as 100% of the user now belongs to an office, they don't provide useful information

Reply:
Ok. Let's do this in a separate PR.